app/services/licences/view-licence-set-up.service.js

```diff
@@ -5,8 +5,10 @@
  * @module ViewLicenceSetUpService
  */
 
+const FeatureFlagsConfig = require('../../../config/feature-flags.config.js')
```

Comment:
See my comment in the presenter; we can grab this there and save passing it through as a param.